README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+
+Hello, my name is **Dinesh Das Gupta**. I am a Second Year in The College, studying Public Policy and Environmental Studies. I represent the Class of 2022 on College Council and sing with the a cappella group Voices in Your Head (Voices).  Voices released an album last year titled  *Begin Again*.  I am studying from Washington, DC, my hometown. I took a gap year before attending UChicago and during that time I worked for:
```

Comment:
This would be a great place to include a header! Something like `# About Me` or `# About Dinesh Das Gupta`. Just something to let the reader know what the following text is explaining